src/wp-includes/ms-blogs.php

```diff
@@ -500,6 +500,10 @@ function update_blog_option( $id, $option, $value, $deprecated = null ) {
 function switch_to_blog( $new_blog_id, $deprecated = null ) {
 	global $wpdb;
 
+	if ( isset( $deprecated ) ) {
+		_deprecated_argument( __FUNCTION__, '3.0.0' );
```

Comment:
```suggestion
		_deprecated_argument( __FUNCTION__, '3.5.0' );
```
Deprecated in https://core.trac.wordpress.org/changeset/21485.